src/Raven.Server/Documents/ETL/EtlLoader.cs

```diff
@@ -212,6 +220,13 @@ private void HandleChangesSubscriptions()
                             connectionStringNotFound = true;
 
                         break;
+                    case EtlType.S3:
+                        s3Config = config as S3EtlConfiguration;
```

Comment:
Need to think about how we can integrate this with the user. Can we use a script to get the details, as well?
That way they can generate that on the fly. Like we do for backups.